### PR TITLE
Add warning and explanation before running command with sudo

### LINKED ISSUE #921: Can the installer provide some info about why root privileges are needed?
```
$ curl -fsSL get.docksal.io | bash
Password:
```

^ That is fairly scary for lots of people, to just give a random script root access without any idea why it's needed or what it's doing. 

I understand that it's necessary in this case to be able to reliably install fin to usr/local/bin, but could the installer output a little helper text before the prompt that just says something like:

> Root privileges needed to move the fin command to /usr/local/bin, see here for more info: [link to docs]

### PR DESCRIPTION
**Pull request by achekulaev**
Addresses and closes https://github.com/docksal/docksal/issues/921

### CODE REVIEW

File: installer.sh [hunk 1]
```diff
@@ -30,6 +30,20 @@ fi
...
 
+is_sudo_granted () {
+	# -S tells sudo to read password from stdin, -v just does "sudo nothing"
+	# If sudo token is already generated this command will succeed
+	# If no token is vaid, this command will fail since echo feeds empty password into sudo
+	echo | sudo -Sv >/dev/null 2>&1
+}
+
+if ! is_sudo_granted; then
+	echo "ATTENTION: Installer requires administrative privileges to continue with Docksal setup."
```
Comment (dnmurray) [inline installer.sh]: You don't really say why you want sudo access. When I first installed docksal, that was my question: why do you want admin access. I pulled down the source and reviewed it to see what it's going to do. This explanation still doesn't really answer the question "why?"

Comment (achekulaev) [reply installer.sh]: @dnmurray if you can help by proposing a concise, descriptive 1-line explanation that works for any OS I will gladly include it here.

Comment (dnmurray) [reply installer.sh]: How about "...administrative privileges to install the Docksal fin command in /usr/local/bin and continue with...?"

On another note, one of the first things I did with my Mac is install Homebrew. I can't remember if I hacked (sudo chmod) /usr/local, but I own almost all the subdirectories. In other words, I had write access to /usr/local/bin, so sudo wasn't required to install fin to /usr/local/bin.  Maybe extend is_sudo_granted() with a test to see if the user can already write to /usr/local/bin?

Comment (achekulaev) [reply installer.sh]: sudo is needed for more stuff. Namely on mac to modify /etc/exports, /etc/resolver/docksal, restart nfsd, clear dns cache... on linux and mac to add ip to the interface and restart it, that's just off the top of my head

Comment (lmakarov) [reply installer.sh]: I added a link to the setup docs in the message. We can cover the "why sudo?" topic there.

**Review by lmakarov — APPROVED** [review]

**Comment by achekulaev** [comment]
Shoot I raised it against the master

**Pull request merged**